modules/navier_stokes/include/fvbcs/WCNSFVSwitchableInletVelocityBC.h

```diff
@@ -0,0 +1,32 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "WCNSFVInletVelocityBC.h"
+
+/**
+ * Dirichlet boundary conditions for the velocity, set from either a velocity postprocessor
+ * or a mass flow rate divided by density and surface
```

Comment:
This is just a copy of the base class doxy I assume. I don't know if you want to add anything about its "switchability"